File: csharp/Gilded-rose.Test/GildedRoseTest.cs
using System.Collections.Generic;
using Xunit;

namespace Gilded_rose.Test
{
    public class GildedRoseTest
    {
        [Fact]
        public void fix_me()
        {
            IList<Item> items = new List<Item> { new Item("foo", 0, 0) };
            var app = new GildedRose(items);

            app.UpdateQuality();

            Assert.Equal("fixme", items[0].Name);
        }

    }
}

File: java/src/test/java/com/gildedrose/GildedRoseTest.java
package com.gildedrose;

import static org.junit.jupiter.api.Assertions.assertEquals;

import org.junit.jupiter.api.Test;

public class GildedRoseTest {

  @Test
  public void fix_me() {
    Item[] items = new Item[]{new Item("foo", 0, 0)};
    GildedRose app = new GildedRose(items);

    app.updateQuality();

    assertEquals("fixme", app.items[0].name);
  }
}
